toolkit/offchain/src/csl.rs

```diff
@@ -150,6 +166,38 @@ pub(crate) fn get_first_validator_budget(
 	Ok(convert_ex_units(&validator_budget.budget))
 }
 
+#[derive(Debug, Clone)]
+pub struct ScriptExUnits {
```

Comment:
How confident are you that indexes in response are always continous?

Reply:
I checked the Ogmios API docs and didn't find any info on that, but considering it's the index of the validator in the transaction, they should